server/log/main.go

```diff
@@ -101,9 +139,17 @@ func main() {
 
 	// Start the sequencing loop, which will run until we terminate the process
 	// TODO(Martin2112): Should respect read only mode and the flags in tree control etc
-	sequencerManager := server.NewSequencerManager(done, simpleMySqlStorageProvider, *batchSizeFlag, *sleepBetweenLogsFlag, *sleepBetweenRunsFlag)
+	// TODO(Martin2112): Plug in Key manager and load key, this is in another branch atm
+	// this is OK as we haven't added code to create a signer task yet
+	sequencerManager := server.NewLogOperationManager(done, simpleMySqlStorageProvider, *batchSizeFlag, *sequencerSleepBetweenRunsFlag, util.SystemTimeSource{}, server.NewSequencerManager(keyManager))
 	go sequencerManager.OperationLoop()
 
+	// Start the signer loop, which will run continuously
```

Comment:
Yes, this isn't the final version of this code. I can't do everything at once :-)